pkgs/games/airshipper/default.nix

```diff
@@ -68,7 +63,7 @@ rustPlatform.buildRustPackage rec {
         libXcursor
       ];
     in
-    ''
+      ''
```

Comment:
this doesn't seem right
```suggestion
    ''
```

Reply:
You are right, indentation problems on my part